components/filter/README.md

```diff
@@ -1,51 +1,193 @@
+	Multi-Dimensional
+	<d2l-filter>
+		<d2l-filter-dimension-set key="role" text="Role">
+			<d2l-filter-dimension-set-value key="admin" text="Admin"></d2l-filter-dimension-set-value>
+			<d2l-filter-dimension-set-value key="student" text="Student" selected></d2l-filter-dimension-set-value>
+			<d2l-filter-dimension-set-value key="teacher" text="Teacher"></d2l-filter-dimension-set-value>
+		</d2l-filter-dimension-set>
+		<d2l-filter-dimension-set key="department" text="Department" selection-single>
+			<d2l-filter-dimension-set-value key="english" text="English"></d2l-filter-dimension-set-value>
+			<d2l-filter-dimension-set-value key="history" text="History" selected></d2l-filter-dimension-set-value>
+			<d2l-filter-dimension-set-value key="math" text="Math" selected></d2l-filter-dimension-set-value>
+			<d2l-filter-dimension-set-value key="science" text="Science"></d2l-filter-dimension-set-value>
+		</d2l-filter-dimension-set>
+	</d2l-filter>
+</div>
+``` 
+
+### Accessibility
+The filter will announce changes to filter selections, search results, and when filters are being cleared. It is up to the consumer to then announce when these changes have propagated and resulted in new/loaded/updated data on the page. This is very important for screenreader users who are not able to visually see the page changing behind the filter control as selections are made.
+
+<!-- docs: start hidden content -->
+### Properties
 
 | Property | Type | Description |
-|--|--|--|
+|---|---|---|
 | `disabled` | Boolean, default: `false` | Disables the dropdown opener for the filter |
 
-**Events:**
+### Events
 * `d2l-filter-change`: dispatched when any filter value has changed (may contain info about multiple dimensions and multiple changes in each)
 * `d2l-filter-dimension-first-open`: dispatched when a dimension is opened for the first time (if there is only one dimension, this will be dispatched when the dropdown is first opened)
 * `d2l-filter-dimension-search`: dispatched when a dimension that supports searching and has the "manual" search-type is searched
 
-**Accessibility**
-The filter will announce changes to filter selections, search results, and when filters are being cleared. It is up to the consumer to then announce when these changes have propogated and resulted in new/loaded/updated data on the page. This is very important for screenreader users who are not able to visually see the page changing behind the filter control as selections are made.
+<!-- docs: end hidden content -->
 
-## Filter Dimension Types
-
-### d2l-filter-dimension-set and d2l-filter-dimension-set-value
+## Filter Dimension Types [d2l-filter-dimension-set] 
 
 The `d2l-filter-dimension-set` component is the main dimension type that will work for most use cases.  Used alongside the `d2l-filter-dimension-set-value`, this will give you a selectable list of filter values.
 
+<!-- docs: demo live name:d2l-filter-dimension-set align:start autoSize:false size:large -->
```

Comment:
I didn't think that the `d2l-filter-dimension-set-value` properties warranted its own demo. It would really just be for testing the selection (which you can do anyway).

Reply:
Fair enough - so then should the properties table for it be visible in Daylight?

Reply:
I played around with this in a couple different formats this morning. I think leaving the properties out might be ideal for this case. 

If we include the markdown table directly, the headers look odd since we would need to use an h3 + h4. If we add the live version of the demo its also weird since the demo is kinda useless, any time you toggle something the filter closes (and the only visual thing is the selection property).

I pushed a slight change to the titles here that may clarify the components a bit

Reply:
I'll move the text to the top, that deals with the heading levels well I think! I'll include the markdown version of the properties table too for `d2l-filter-dimension-set-value`. 

I think Stacey already added `Required` to all the properties that need it.

Reply:
Ah perfect! Sounds good